kuma/wiki/migrations/0011_auto_20210108_0416.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2021-01-08 04:16
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    Flag = apps.get_model("waffle", "Flag")
+    Flag.objects.filter(
+        name__in=["kumaediting", "page_move", "section_edit", "spam_checks_enabled"]
```

Comment:
This should have been done in `0010_auto_20210107_1338.py` but I failed to actually call the `forward` function.